### ISSUE TITLE
Seasoning Rack - texture issue

### ISSUE THREAD

**Issue by jesmores** [GlobbyPotato/Rockhounding_Chemistry#66]
![2018-07-19_09 13 55](https://user-images.githubusercontent.com/125214/42956268-65b9a424-8b34-11e8-9b4d-fe96b0e1796e.png)

It's a little hard to show with just a screenshot, but the texture inside the slats/layers is flickering when you move around. z-fighting maybe?

**Comment by GlobbyPotato** [comment]
Yep, definitely a glitch, will fixed in the next upload, Thanks for reporting it.
